### PR TITLE
Add users and teams to dependabot security alert information

### LINKED ISSUE #4760: Provide instructions for how to allow non admins to see dependabot security alerts
### What article on docs.github.com is affected?

https://docs.github.com/en/code-security/supply-chain-security/viewing-and-updating-vulnerable-dependencies-in-your-repository

### What part(s) of the article would you like to see updated?

`Viewing and updating vulnerable dependencies`

Doesn't explain how a non admin can view security alerts, it appears to assume that everyone is an admin of all repositories.

There is no documentation for an admin explaining how to let non admins see the security alerts.

This leads to two problems:
1. Admins see alerts and don't understand why others can't see them
2. Others get urls that don't work and don't understand what to do

### Additional information

The current public facing information for this is here:
https://github.community/t/cant-see-the-access-to-alerts-for-dependabot-settings/157526

It does not include pictures, it isn't maintained, and it is not discoverable.

---

### Content design plan

1. Change to the [Viewing and updating vulnerable dependencies in your repository](https://docs.github.com/en/code-security/supply-chain-security/managing-vulnerabilities-in-your-projects-dependencies/viewing-and-updating-vulnerable-dependencies-in-your-repository) article:

    - In the metadata at the top of the file, update the `permissions` statement. It should mention users with explicit access to security alerts.

2. Change to the reusable that defines step 3 of the procedure.
    - After the text, before the `{% ifversion ftp %}` condition, add a brief sentence explaining that if the option is missing, they should ask for access to security alerts. Then add a "For more information..." link to [Managing security and analysis settings for your repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-security-and-analysis-settings-for-your-repository#granting-access-to-security-alerts)

### PR DESCRIPTION
**Pull request by aisgbnok**
### Why:

Closes #4760

### What's being changed:

[Viewing and updating Dependabot alerts](https://docs.github.com/code-security/dependabot/dependabot-alerts/viewing-and-updating-dependabot-alerts):

1. Permissions metadata now includes users and teams that have explicit access.
2. `sidebar-dependabot-alerts.md` reusable now informs users of the security permission requirement for dependabot alerts and includes a link to [Managing security and analysis settings for your repository](https://docs.github.com/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-security-and-analysis-settings-for-your-repository#granting-access-to-security-alerts).

Permissions (BEFORE)
image
Permissions (AFTER)
image

`sidebar-dependabot-alerts.md` (BEFORE)
image
`sidebar-dependabot-alerts.md` (AFTER)
image

If anyone has any suggestions or improvements, such as wording or grammar, please let me know!

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [X] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@aisgbnok 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by mchammer01** [comment]
I'll review this, today or tomorrow 🙂

File: content/code-security/dependabot/dependabot-alerts/viewing-and-updating-dependabot-alerts.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ redirect_from:
   - /github/managing-security-vulnerabilities/viewing-and-updating-vulnerable-dependencies-in-your-repository
   - /code-security/supply-chain-security/viewing-and-updating-vulnerable-dependencies-in-your-repository
   - /code-security/supply-chain-security/managing-vulnerabilities-in-your-projects-dependencies/viewing-and-updating-vulnerable-dependencies-in-your-repository
-permissions: Repository administrators and organization owners can view and update dependencies.
+permissions: Repository administrators and organization owners can view and update dependencies as well as users and teams with explicit access.
```
Comment (mchammer01) [inline content/code-security/dependabot/dependabot-alerts/viewing-and-updating-dependabot-alerts.md:9]: Nit, adding a comma as this is an end-sentence parenthesis 
```suggestion
permissions: Repository administrators and organization owners can view and update dependencies, as well as users and teams with explicit access.
```

**Review by mchammer01 — COMMENTED** [review]
@aisgbnok - thanks for improving our docs, LGTM ✨ 
Just a couple of minor suggestions that I'm going to apply.
Thank you again for your contribution 🥇

**Review by mchammer01 — APPROVED** [review]
Approving 👍🏻

**Pull request merged**